.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,46 @@
+name: Update WPT
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
A comment saying how often this is would be nice

Reply:
thanks